build-system/tasks/dist.js

```diff
@@ -25,7 +25,7 @@ const {
   compileCoreRuntime,
   compileJs,
   endBuildStep,
-  maybeToEsmName,
+  maybeToEsmOrSxgName,
```

Comment:
Not a huge fan of this name now that there's more than one alternative... how about something like `chooseExtension`? // cc: @rsimha

Reply:
+1 to @danielrozenberg in case we have more flavors in the future

Reply:
Agree with the first comment. However, I was confused by `chooseExtension` because it sounded like it modified the name of the _AMP_ extension and not the _file_ extension 😃

I think this should be called `getMinifiedName()`. That way, the code will read:

```js
minifiedName: getMinifiedName('experiments.js'),
```

Reply:
Done.